package.json

```diff
@@ -70,6 +70,6 @@
     "typescript": "^4.2.4"
   },
   "engines": {
-    "node": "~14.16"
+    "node": "18.18.0"
```

Comment:
Should you do `~18.18`?

Reply:
18.18.0 is the LTS version, I think it's safe enough to lock down this version everywhere.

Reply:
I actually changed it to `>=14.16` so that we can use this package for anything.
There is no actual requirement for this package to use either version, but corona won't build if we limit it to 14.16